api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -431,34 +429,18 @@ public void validateAllOfUsTermsOfService(Integer tosVersion) {
     }
   }
 
-  // Returns true if user has accepted the latest AoU Terms of Service Version
   @Override
-  public boolean validateAllOfUsTermsOfServiceVersion(@Nonnull DbUser dbUser) {
+  public boolean hasSignedLatestAoUTermsOfService(@Nonnull DbUser dbUser) {
     return userTermsOfServiceDao
         .findFirstByUserIdOrderByTosVersionDesc(dbUser.getUserId())
         .map(u -> u.getTosVersion() == configProvider.get().termsOfService.latestAouVersion)
         .orElse(false);
   }
 
-  // Returns true only if the user has accepted the latest version of both AoU and Terra terms of
-  // service
   @Override
-  public boolean validateTermsOfService(@Nonnull DbUser dbUser) {
-    return validateAllOfUsTermsOfServiceVersion(dbUser) && getUserTerraTermsOfServiceStatus(dbUser);
-  }
-
-  @Override
-  public boolean getUserTerraTermsOfServiceStatus(@Nonnull DbUser dbUser) {
```

Comment:
the new version doesn't throw an ApiException, so it can be reduced to one line.  so: I inlined it.